### DISCUSSION TITLE
I want to setup flower behind a proxy which is unaware of proxy

### DISCUSSION THREAD

**Discussion by VikalpRusia** [mher/flower#1359] [Q&A]
Example -> example.com/flower
Reverse proxy rewrites this url via removing flower and sending to our flower instance but css etc. doesn't load

If we use --url_prefix this causes flower instance to expect urls with /flower while our reverse proxy overrides it

I want to have only css (static files) to be have /flower appened 
**A path prefix handled by a proxy that is not seen by the application but is seen by external clients**

**Comment by manviii27** [comment]
Idk if I am understanding this correctly. But you want a proxy that will redirect requests to flower, is that it?
